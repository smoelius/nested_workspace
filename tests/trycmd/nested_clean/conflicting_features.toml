args = ["nested", "clean"]

[bin]
name = "cargo-nested"

[fs]
cwd = "../../../fixtures/conflicting_features"
