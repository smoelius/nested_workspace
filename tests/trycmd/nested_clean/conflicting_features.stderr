     Removed [..] files[..]
<<< [CWD]/big_endian
     Removed [..] files[..]
>>> [CWD]/big_endian
<<< [CWD]/little_endian
     Removed [..] files[..]
>>> [CWD]/little_endian
<<< [CWD]/to_bytes
     Removed [..] files[..]
>>> [CWD]/to_bytes
